api/requirements_dev.in

```diff
@@ -10,4 +10,4 @@ pip-tools
 
 # Useful extra developer packages:
 factory-boy
-locustio
+locust
```

Comment:
Same as the above commit; preferably try to keep changes like this in a single commit, and inside that commit explain why you are doing this. 

For me as a dev external to the project it's quite unclear - although I believe there are proper reasons :)